helm-service/controller/handler_base.go

```diff
@@ -58,6 +61,30 @@ func (h *HandlerBase) existsGeneratedChart(e keptnv2.EventData) (bool, error) {
 	return helm.DoesChartExist(e, genChartName, h.getConfigServiceURL())
 }
 
+func (h *HandlerBase) getCustomDeploymentURLs(chart *chart.Chart) (string, string) {
```

Comment:
please add a unit test for this function. Also in this case, it doesn't have to be a pointer receiver

Reply:
Two remarks: 
1. The current implementation will `panic` if the value is of type other than `string` 
2. We should avoid testing **unexported** functions, these are implementation details and could change anytime. As an alternative, you could provide a more generic form of this function (e.g. `func GetValueAsStr(chart chart.Chart , val string) (string,error)` in `helm_helper.go`, test it via unit tests, and make use of it here in `handler_base.go`.